crypto/asn1/x_long.c

```diff
@@ -56,11 +56,30 @@ static void long_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
     *(long *)pval = it->size;
 }
 
+static int num_bits_ulong(unsigned long value)
```

Comment:
*shrug*

I'm looking forward a few years, and someone wondering why we have several functions basically providing the same function.  A bit like we've had a few variants of hex2str around in the code and recently consolidated that into one...

If you think there's good reason to recreate a similar situation, sure, fine, but please comment on it somehow, or there will be someone consolidating them a few years from now.